pylabrobot/resources/carrier.py

```diff
@@ -243,32 +250,35 @@ def __init__(
       sites,category=category, model=model)
 
 
-class ShakerCarrier(Carrier):
-  """ Base class for shaker carriers (i.e. 7-track carriers with mixed-use and/or specialized
-  sites). """
+class TubeCarrier(Carrier):
+  """ Base class for tube/sample carriers.
+  Name prefix: 'SMP_'
+  """
   def __init__(
     self,
     name: str,
     size_x: float,
     size_y: float,
     size_z: float,
     sites: Optional[List[CarrierSite]] = None,
-    category="shaker_carrier",
+    category="tube_carrier",
     model: Optional[str] = None):
     super().__init__(name, size_x, size_y, size_z,
       sites,category=category, model=model)
 
 
-class TubeCarrier(Carrier):
-  """ Base class for tube/sample carriers. """
+class ReagentCarrier(Carrier):
+  """ Base class for reagent/trough carriers.
+  Name prefix: 'RGT_'
```

Comment:
I thought this would be the Trough carrier. (all Carriers can have reagents)

Reply:
Yes, I completely agree. Should we still keep the "RGT_" name prefix for instances though?

Reply:
I am fully switched to abandoning this manure argument and switching to first-principle reasoning wherever possible. Can we use "TGH" or even just "Trough" or similar?

Reply:
Yes, I would like that too. Simply `Trough_` sounds the simplest to me.

I only copied the prefixes directly over because that was the last consensus on the [labautomation forum thread discussion](https://labautomation.io/t/standardized-labware-naming-convention-discussion/3787/14). Could you please message your proposal for an alternative there.

I am happy to change this to docstring to "Trough_" to finish this PR and update if things change.

Reply:
merging this now, we can update the doc string later. It will is inconsequential in this timeframe.